frontend/src/api/products.js

```diff
@@ -0,0 +1,83 @@
+const products = [
+    {
+        id: '1',
+        name: 'Canon EOS 1500D 24.1 Digital SLR Camera (Black) with EF S18-55 is II Lens',
+        image: 'https://m.media-amazon.com/images/I/914hFeTU2-L._AC_UY218_.jpg',
+        description:
+            '1 Camera, Eyecup Ef, Body Cap Battery Charger LC-E10E Battery Pack LP-E10 Interface Cable Wide Strap EW-400D',
+        category: 'Electronics',
+        price: 30000,
+        countInStock: 10,
+        rating: 4.5,
+        numReviews: 12,
+        qty: 0,
+
+    },
+    {
+        id: '2',
+        name: 'Right Choice Polyester Picnic Hiking Camping Portable Dome Tent with bag for 4 person',
+        image: 'https://images-na.ssl-images-amazon.com/images/I/51CAlmnPROL._SY450_.jpg',
+        description:
+            "Tent Size : 200 X 200 X 135 cm In The Box 1 x Picnic hiking camping portable dome tent for 4 person waterproof with bag Good for outdoor like picnic",
+        category: 'Trekking',
+        price: 1400,
+        countInStock: 7,
+        rating: 4.0,
+        numReviews: 8,
+        qty: 0
+    },
+    {
+        id: '3',
+        name: 'Skechers Women Go Run Focus-Approach Shoes',
+        image: 'https://m.media-amazon.com/images/I/71s16iLzhaL._AC_UL320_.jpg',
+        description:
+            'Rubber sole sketchers shoes for women with round toe style.',
+        category: 'Shoes',
+        price: 3000,
+        countInStock: 5,
+        rating: 3,
+        numReviews: 12,
+        qty: 0
+    },
+    {
+        id: '4',
+        name: 'Skybags Mint 80 cms Polycarbonate Turquoise Hardsided Check-in Luggage',
+        image: 'https://images-na.ssl-images-amazon.com/images/I/51O%2Bw3BDAhL._SX425_.jpg',
+        description:
+            'Fixed combination lock for added security Country of Origin: India Extremely lightweight bags for carrying ease',
+        category: 'Bags',
+        price: 5000,
+        countInStock: 11,
+        rating: 5,
+        numReviews: 12,
+        qty: 0
+    },
+    {
+        id: '5',
+        name: 'POLESTAR XPLORE 55 Ltrs Rucksack Hiking Backpack with Rain Cover (Blue)',
+        image: 'https://images-na.ssl-images-amazon.com/images/I/817m19TT%2BKL._SX522_.jpg',
+        description:
+            'RAIN COVER & WATER PROOF BASE : a rain cover can make hiking in bad weather as enjoyable as hiking in good weather.',
+        category: 'Bags',
+        price: 1000,
+        countInStock: 7,
+        rating: 3.5,
+        numReviews: 10,
+        qty: 0
+    },
+    {
+        id: '6',
+        name: 'AFEO Fabric Shoe Bag (Black_GR-PVC-BLACK_12)',
+        image: 'https://images-na.ssl-images-amazon.com/images/I/71BrIW0BA0L._SX522_.jpg',
+        description:
+            'Outer & Inner Material: Fabric (Non Woven) Color: Black Water Resistant: Yes',
+        category: 'Extra',
+        price: 300,
+        countInStock: 0,
```

Comment:
![image](https://user-images.githubusercontent.com/44545218/113938255-f3fc4c80-9817-11eb-82ee-e6a7dd02232a.png)
This has lead to this particular ui issue , do check it out when you get time

Reply:
![image](https://user-images.githubusercontent.com/60965304/114259602-ba306f00-99ec-11eb-82f6-c60c8b2a4c52.png)
doesn't shows in mine.